docs/ABOUT.md

```diff
@@ -1,14 +1,5 @@
+R is an open source programming language for data analysis and statistical computing. It was created by Ross Ihaka and Robert Gentleman at the University of Auckland, and is an implementation of the S programming language combined with lexical scoping semantics inspired by Scheme. The language is interpreted, dynamically typed, and has a strong foundation in functional programming. This makes it well suited to many of the challenges of data analysis.
 
-R provides a wide variety of statistical (linear and nonlinear modelling, classical statistical tests, time-series analysis, classification, clustering, etc) and graphical techniques, and is highly extensible.
+It's worth noting however, that R is not just a programming language, but also an environment for interactive data analysis. Its powerful metaprogramming capabilities allow for magically succinct and concise functions. It has also been designed to leverage high-performance programming languages like C, Fortran, and C++. These combined attributes provide an excellent environment for designing domain-specific languages.
 
-One of its strengths is the ease with which well-designed publication-quality plots can be produced, including mathematical symbols and formulae where needed. Great care has been taken over the defaults for the minor design choices in graphics, but the user retains full control. Along with this, for computationally-intensive tasks, C, C++ and Fortran can be linked and called at runtime or one can even write C code to manipulate R objects directly.
-
-The R software suite includes:
-* An effective data handling and storage facility,
-* A suite of operators for calculations on arrays, in particular matrices,
-* A large, coherent, integrated collection of intermediate tools for data analysis,
-* graphical facilities for data analysis and display either on-screen or on hardcopy, and
-* A well-developed, simple and effective programming language which includes conditionals, loops, user-defined recursive functions and input and output facilities.
-* The term “environment” is intended to characterize it as a fully planned and coherent system, rather than an incremental accretion of very specific and inflexible tools, as is frequently the case with other data analysis software.
-* R can be extended (easily) via packages. There are about eight packages supplied with the R distribution and many more are available through the CRAN family of Internet sites covering a very wide range of modern statistics.
+R has a fantastic community of both users and developers, who contribute to a massive (and rapidly growing) collection of packages for importing, manipulating and visualising data, as well as cutting edge methods for statistical modelling and machine learning.
```

Comment:
This is really lovely as well. Both of these last sections make me want to start learning R.

I really like that you shortened the ABOUT doc and got rid of the bullet list. This feels a lot friendlier and more enticing to me.